spec/views/report/_form_expression_buttons.html.haml_spec.rb

```diff
@@ -0,0 +1,27 @@
+describe 'report/_form_filter.html.haml' do
+  before :each do
+    report = FactoryGirl.create(:miq_report)
+    @expkey = :record_filter
+    exp_table = [["Example AFTER \"value1\"", 1]]
+    expression = {"after" => {"field" => "field_example", "value" => "value1"}, :token => 1}
+    @edit = {:rpt_id => report.id,
+             @expkey => {:exp_table => exp_table, :expression => expression}}
+  end
+
+  it 'renders form expression buttons for other report and Edit Display Filter button' do
+    render :partial => 'report/form_expression_buttons',
```

Comment:
When you're testing a view using `render`, you should use [`assign`](https://relishapp.com/rspec/rspec-rails/v/2-1/docs/view-specs/view-spec) for setting the `@` instance variables.